pex/variables.py

```diff
@@ -33,11 +33,46 @@ def iter_help(cls):
       variable_type, variable_text = cls.process_pydoc(getattr(value, '__doc__'))
       yield variable_name, variable_type, variable_text
 
+  @classmethod
+  def from_rc(cls, rc='~/.pexrc', build_time_rc_dir=None):
+    """Read pex runtime configuration variables from a pexrc file.
+
+    :param rc: an absolute path to a pexrc file
+    :param build_time_rc_dir: an absolute path to a directory containing a pexrc. This is needed to
+    read a pexrc at build time in the case where a pexrc is located in the same directory as a the
+    output pex.
+
+    :return ret_vars: a dict object containing key values pairs found in processed pexrc files
+    """
+    ret_vars = {}
+    rc_locations = ['/etc/pexrc',
+                    rc,
+                    os.path.join(os.path.dirname(sys.argv[0]), '.pexrc'),
+                    os.getcwd()]
```

Comment:
afaict, this should reference an rcfile, not the current working dir directly?

but maybe it's not actually needed?

Reply:
I think it is needed for the case where the pex you are invoking and the .pexrc is in the cwd.
Also, idk why the `pi = interpreter or PythonInterpreter.get()` line was already changed to that, I am not sure why this reverted, but I will re-add, sorry about that.